protractor-jasmine-tests/conf.js

```diff
@@ -0,0 +1,70 @@
+const SpecReporter = require('jasmine-spec-reporter').SpecReporter;
+exports.config = {
+    /*sauceUser: process.env.SAUCE_USERNAME,
+    sauceKey: process.env.SAUCE_ACCESS_KEY,*/
+    sauceRegion: 'us',
+    seleniumAddress: 'https://ondemand.saucelabs.com/wd/hub',
+    specs: ['specs/*spec.js'],
+
+    multiCapabilities: [{
+        browserName: 'chrome',
+        version: 'latest',
+        platform: 'Windows 10',
+        name: 'chrome-protractor-test',
+        build: 'Sample Protractor Tests',
+        username: process.env.SAUCE_USERNAME,
+        accessKey: process.env.SAUCE_ACCESS_KEY,
+        shardTestFiles: true,
+        maxInstances: 25
+    },{
+        browserName: 'firefox',
+        version: 'latest',
+        platform: 'Windows 10',
+        name: 'firefox-protractor-test',
+        build: 'Sample Protractor Tests',
+        username: process.env.SAUCE_USERNAME,
```

Comment:
@spider-sauce 

This is not the recommended way to do it, like we discussed yesterday